projects/common/src/tokens/crypto.ts

```diff
@@ -0,0 +1,9 @@
+import {inject, InjectionToken} from '@angular/core';
+import {WINDOW} from './window';
+
+export const CRYPTO = new InjectionToken<Crypto>(
+    'An abstraction over window.crypto object',
+    {
+        factory: () => inject(WINDOW).localStorage,
```

Comment:
You forgot to change this 🙂

Reply:
Woops, indeed. copy/pasted too quickly :)

Reply:
Not sure of github sends notifications for force-pushes. Updated, though.